src/components/OptionsList/types.ts

```diff
@@ -0,0 +1,138 @@
+import type {RefObject} from 'react';
+import type {SectionList, SectionListData, StyleProp, View, ViewStyle} from 'react-native';
+import type {OptionData} from '@libs/ReportUtils';
+
+type Section = {
+    /** Title of the section */
+    title: string;
+
+    /** The initial index of this section given the total number of options in each section's data array */
+    indexOffset: number;
+
+    /** Array of options */
+    data: OptionData[];
+
+    /** Whether this section should show or not */
+    shouldShow?: boolean;
+
+    /** Whether this section is disabled or not */
+    isDisabled?: boolean;
+};
+
+type SelectedOptionData = OptionData & {
+    /** The name of selected option */
+    name: string;
+};
```

Comment:
It is created because selected options use an extra `name` prop:
https://github.com/Expensify/App/blob/dc3e19bdcc95023d74cd7fdd433f6208707eaaf8/src/components/CategoryPicker/index.js#L28

Reply:
Why not add it to OptionData instead?